{
  "type": "issue",
  "issue": {
    "id": 89641981,
    "node_id": "MDU6SXNzdWU4OTY0MTk4MQ==",
    "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/6311",
    "repository_url": "https://api.github.com/repos/bitcoin/bitcoin",
    "labels_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/6311/labels%7B/name%7D",
    "comments_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/6311/comments",
    "events_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/6311/events",
    "html_url": "https://github.com/bitcoin/bitcoin/issues/6311",
    "number": 6311,
    "state": "closed",
    "state_reason": "completed",
    "title": "Error initializing wallet database environment on fresh install",
    "body": "Hello,\n\nIm having an issue setting up dogecoin in ubuntu 14.04 for a test environment \n\nI know the build will not be portable because i installed libdb5.1 from the ubuntu ppa. But i cant even get the daemon to start\n\nError initializing wallet database environment /home/doge/.dogecoin!\n\nEven tried removing .dogecoin and letting the daemon generate it. Still get the same error. \n",
    "user": {
      "login": "Mullick",
      "id": 5300040,
      "node_id": "MDQ6VXNlcjUzMDAwNDA=",
      "avatar_url": "https://avatars.githubusercontent.com/u/5300040?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/Mullick",
      "html_url": "https://github.com/Mullick",
      "followers_url": "https://api.github.com/users/Mullick/followers",
      "following_url": "https://api.github.com/users/Mullick/following%7B/other_user%7D",
      "gists_url": "https://api.github.com/users/Mullick/gists%7B/gist_id%7D",
      "starred_url": "https://api.github.com/users/Mullick/starred%7B/owner%7D%7B/repo%7D",
      "subscriptions_url": "https://api.github.com/users/Mullick/subscriptions",
      "organizations_url": "https://api.github.com/users/Mullick/orgs",
      "repos_url": "https://api.github.com/users/Mullick/repos",
      "events_url": "https://api.github.com/users/Mullick/events%7B/privacy%7D",
      "received_events_url": "https://api.github.com/users/Mullick/received_events",
      "type": "User",
      "site_admin": false
    },
    "labels": [],
    "assignees": [],
    "author_association": "NONE",
    "locked": true,
    "active_lock_reason": "resolved",
    "comments": 1,
    "closed_at": "2015-06-19T19:42:49Z",
    "created_at": "2015-06-19T19:32:39Z",
    "updated_at": "2021-09-08T12:34:00Z"
  },
  "events": [
    {
      "event": "commented",
      "id": 113620275,
      "node_id": "MDEyOklzc3VlQ29tbWVudDExMzYyMDI3NQ==",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/113620275",
      "actor": {
        "login": "Mullick",
        "id": 5300040,
        "node_id": "MDQ6VXNlcjUzMDAwNDA=",
        "avatar_url": "https://avatars.githubusercontent.com/u/5300040?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Mullick",
        "html_url": "https://github.com/Mullick",
        "followers_url": "https://api.github.com/users/Mullick/followers",
        "following_url": "https://api.github.com/users/Mullick/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Mullick/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Mullick/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Mullick/subscriptions",
        "organizations_url": "https://api.github.com/users/Mullick/orgs",
        "repos_url": "https://api.github.com/users/Mullick/repos",
        "events_url": "https://api.github.com/users/Mullick/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Mullick/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2015-06-19T19:42:47Z",
      "updated_at": "2015-06-19T19:42:47Z",
      "author_association": "NONE",
      "body": "First off wrong repository. Sorry about that\n\nSecondly it looks like the issue was on my end.\n\napt-get autoremove removed some old libdb 5.1 packages which im guessing dogecoin used to build as after it spewed an error it couldnt find the libdb files\n\nRecomplied and it worked. \n",
      "user": {
        "login": "Mullick",
        "id": 5300040,
        "node_id": "MDQ6VXNlcjUzMDAwNDA=",
        "avatar_url": "https://avatars.githubusercontent.com/u/5300040?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Mullick",
        "html_url": "https://github.com/Mullick",
        "followers_url": "https://api.github.com/users/Mullick/followers",
        "following_url": "https://api.github.com/users/Mullick/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Mullick/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Mullick/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Mullick/subscriptions",
        "organizations_url": "https://api.github.com/users/Mullick/orgs",
        "repos_url": "https://api.github.com/users/Mullick/repos",
        "events_url": "https://api.github.com/users/Mullick/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Mullick/received_events",
        "type": "User",
        "site_admin": false
      },
      "html_url": "https://github.com/bitcoin/bitcoin/issues/6311#issuecomment-113620275",
      "issue_url": "https://api.github.com/repos/bitcoin/bitcoin/issues/6311"
    },
    {
      "event": "closed",
      "id": 335566010,
      "node_id": "MDExOkNsb3NlZEV2ZW50MzM1NTY2MDEw",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/335566010",
      "actor": {
        "login": "Mullick",
        "id": 5300040,
        "node_id": "MDQ6VXNlcjUzMDAwNDA=",
        "avatar_url": "https://avatars.githubusercontent.com/u/5300040?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Mullick",
        "html_url": "https://github.com/Mullick",
        "followers_url": "https://api.github.com/users/Mullick/followers",
        "following_url": "https://api.github.com/users/Mullick/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/Mullick/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/Mullick/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/Mullick/subscriptions",
        "organizations_url": "https://api.github.com/users/Mullick/orgs",
        "repos_url": "https://api.github.com/users/Mullick/repos",
        "events_url": "https://api.github.com/users/Mullick/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/Mullick/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2015-06-19T19:42:49Z"
    },
    {
      "event": "locked",
      "id": 5272010479,
      "node_id": "LOE_lADOABII584FV9P9zwAAAAE6PH7v",
      "url": "https://api.github.com/repos/bitcoin/bitcoin/issues/events/5272010479",
      "actor": {
        "login": "DrahtBot",
        "id": 39886733,
        "node_id": "MDQ6VXNlcjM5ODg2NzMz",
        "avatar_url": "https://avatars.githubusercontent.com/u/39886733?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/DrahtBot",
        "html_url": "https://github.com/DrahtBot",
        "followers_url": "https://api.github.com/users/DrahtBot/followers",
        "following_url": "https://api.github.com/users/DrahtBot/following%7B/other_user%7D",
        "gists_url": "https://api.github.com/users/DrahtBot/gists%7B/gist_id%7D",
        "starred_url": "https://api.github.com/users/DrahtBot/starred%7B/owner%7D%7B/repo%7D",
        "subscriptions_url": "https://api.github.com/users/DrahtBot/subscriptions",
        "organizations_url": "https://api.github.com/users/DrahtBot/orgs",
        "repos_url": "https://api.github.com/users/DrahtBot/repos",
        "events_url": "https://api.github.com/users/DrahtBot/events%7B/privacy%7D",
        "received_events_url": "https://api.github.com/users/DrahtBot/received_events",
        "type": "User",
        "site_admin": false
      },
      "commit_id": null,
      "commit_url": null,
      "created_at": "2021-09-08T12:34:00Z",
      "lock_reason": "resolved"
    }
  ]
}